### PR TITLE
Fix various issues with the Docker dev env setup

### PR DESCRIPTION
**Pull request by underyx**

### CODE REVIEW

File: pyconcz_2016/settings.py [hunk 1]
```diff
@@ -83,18 +83,21 @@
...
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'HOST': os.environ['RDS_HOST'],
-        'NAME': os.environ.get('RDS_NAME', RDS_USER),
-        'USER': RDS_USER,
-        'PASSWORD': os.environ['RDS_PASSWORD'],
-        'PORT': os.environ.get('RDS_POST', 5432),
+try:
+    RDS_USER = os.environ['RDS_USER']
```
Comment (tricoder42) [inline pyconcz_2016/settings.py:87]: RDS_USER and RDS_HOST are both required, so these errors shouldn't be silenced. Empty `DATABASES` config shouldn't be allowed in production settings.

**Comment by tricoder42** [comment]
Thanks for this fix! I didn't rebuild container since I changed file structure :(

**Comment by tricoder42** [comment]
I'll merge it in the morning without the last commit. In development app doesn't require RDS settings (using `pyconcz_2016.settings_dev`).

**Comment by underyx** [comment]
@tricoder42 in response to the `RDS_USER` being required: I thought that would be the case as well, but when you're using the dev settings module, it imports the plain `settings.py` file right at the top. Since the import executes the contents of `settings.py`, this `KeyError` will cause a failure in `dev_settings.py`, even though the database config would be overridden right afterwards.

I spent a short while trying to come up with a nicer solution, but I think the best way is to just accept that the variables in the settings can be overridden, which makes empty values in it reasonable.

File: pyconcz_2016/settings.py (hunk 1 above)
Comment (underyx) [inline pyconcz_2016/settings.py:87]: See response here: https://github.com/pyvec/cz.pycon.org-2016/pull/30#issuecomment-234406074

(this line note didn't show up on mobile for me :dancers:)

**Comment by tricoder42** [comment]
@underyx Ah, you're right! I didn't think about it. Merging, thanks! :)

**Pull request merged**